tool/lib/commands/analyze.dart

```diff
@@ -41,32 +42,20 @@ class AnalyzeCommand extends Command {
 
       final progress = log.progress('  ${p.relativePath}');
 
-      final process = await Process.start(
-        sdk.dartToolPath,
-        ['analyze', '--fatal-infos'],
+      final process = await processManager.runProcess(
+        CliCommand.from(
+          sdk.dartToolPath,
+          ['analyze', '--fatal-infos'],
+        ),
         workingDirectory: p.packagePath,
       );
-      final Stream<List<int>> stdout = process.stdout;
-      final Stream<List<int>> stderr = process.stderr;
 
-      final int exitCode = await process.exitCode;
-
-      if (exitCode == 0) {
+      if (process.exitCode == 0) {
         progress.finish(showTiming: true);
       } else {
```

Comment:
same comment here. We will never hit this else case because an exception would have been thrown by now

Reply:
since the else case was just there to track the error, can we just remove the if / else entirely and then if the process has a non-zero exit code, an exception will be thrown from `processManager.runProcess`?